tests/e2e/unified_test.go

```diff
@@ -376,6 +376,31 @@ func runScenario(ctx context.Context, t *testing.T, testPause bool, fixture reso
 					}
 				}
 
+				if ShouldTestRereconiliation(t, primaryResource) {
```

Comment:
Awesome! We finally have the "nochange" test equivalence in mock test. Thanks a lot!